ckanext/canada/tests/test_travelq.py

```diff
@@ -7,6 +10,8 @@
 
 from ckanext.recombinant.tables import get_chromo
 
+DESTINATION_ENDPOINT = 'https://restcountries.com/v3.1/all?fields=name,capital'
```

Comment:
We have a specific list of countries recognized by the GoC that is maintained with https://github.com/open-data/ckanext-canada/blob/master/bin/download_country.py to add XK and TW

Reply:
@wardi so just use pyyaml and the choices file?? I don't specifically need the capital city names I guess, I could just double up the country name

Reply:
might be better than adding a dependency on another 3rd party service. We could collect the capital names from the same source that script uses if you really want.

Was the purpose to check that your regex covers the characters required to represent common city names?

Reply:
yeah was just trying to do city names as well